### DISCUSSION TITLE
Can I submit a package to the repository even if I am not the owner or a dev of the app?

### DISCUSSION THREAD

**Discussion by thinasdf** [microsoft/winget-cli#4073] [Q&A] [answered]
There are many programs I use that I would like to be able to use winget to install and update. But they are not in the repository. Can I submit them myself or only the developers of them are allowed to do this?

**Comment by mdanish-kh** [comment, marked as answer]
Yes. As long as the publisher has a publicly available download URL, you should be able to create a manifest for it.

> Can I submit them myself or only the developers of them are allowed to do this?

There are some packages whose publishers have restricted the capability of community contributions to some extent. They are defined in this [Auth.csv](https://github.com/microsoft/winget-pkgs/blob/master/Tools/Auth.csv) file. A strictness of `should` will notify the publisher but will not block the approval process (which means you can still submit a PR for that package). Strictness of `must` will block the merge process pending the approval of the publisher of the package. Other than those, you are highly encouraged to submit manifests for the applications you want to see in WinGet!
